3.3/.sti/bin/assemble

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /opt/openshift/etc/generate_container_user
```

Comment:
I don't think we need it for assemble... only for run. When running assemble we just use the user set in the Dockerfile.